From 60ea0b4c264441055cf242079b5c4987430ce396 Mon Sep 17 00:00:00 2001
From: Shubhangi Shrivastava <shubhangi.shrivastava@intel.com>
Date: Mon, 21 Sep 2015 16:51:35 +0530
Subject: [PATCH] SQUASHME! [VPG]: drm: Fix warning in
 drm_mode_obj_set_property_ioctl, drm_crtc.c

Not terminating string in same line generates compilation error
when warnings are treated as errors. This patch resolves this
compilation issue in drivers/gpu/drm/drm_crtc.c:4200:4:
error: missing terminating " character
    DRM_DEBUG_KMS("CRTC from connector,CRTC={id=%d props=

This patch should be squashed into the following patch:

commit 4feccbbe33cf56e0f4e75e58f12c1961d42669b5
Change-Id: I73e94c6541f4621d86371cadd580dcdfc33c50d3
Author: Akashdeep Sharma <akashdeep.sharma@intel.com>
Date:   Mon Jun 1 19:23:30 2015 +0530
    SQUASHME! [VPG]: drm: Use right level of log message in set property ioctl

Change-Id: I3350870975434812e9672043bd3907a3949053c9
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-4749
Signed-off-by: Akashdeep Sharma <akashdeep.sharma@intel.com>
Signed-off-by: Shubhangi Shrivastava <shubhangi.shrivastava@intel.com>
---
 drivers/gpu/drm/drm_crtc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/drm_crtc.c b/drivers/gpu/drm/drm_crtc.c
index 858936d..49e0c8e 100644
--- a/drivers/gpu/drm/drm_crtc.c
+++ b/drivers/gpu/drm/drm_crtc.c
@@ -4197,8 +4197,8 @@ int drm_mode_obj_set_property_ioctl(struct drm_device *dev, void *data,
 	case DRM_MODE_OBJECT_CONNECTOR:
 		crtc = drm_crtc_from_connector(obj_to_connector(arg_obj));
 		if (crtc) {
-			DRM_DEBUG_KMS("CRTC from connector,CRTC={id=%d props=
-				%d}\n", crtc->base.id, crtc->base.properties ?
+			DRM_DEBUG_KMS("CRTC from connector,{id=%d props=%d}\n",
+					crtc->base.id, crtc->base.properties ?
 					crtc->base.properties->count : 0);
 			mutex_lock(&config->mutex);
 			drm_modeset_lock(&crtc->mutex, NULL);
-- 
1.9.1

